nntrainer/utils/util_func.h

```diff
@@ -309,6 +309,26 @@ tm *getLocaltime(tm *tp);
  */
 std::regex getRegex(const std::string &str);
 
+/**
+ * @brief  Convert a floating-point number into its fixed-point component (the
+ * mantissa) and its exponent component.
+ *
+ * @param[in] input floating point to convert
+ * @param[out] fixedpoint fixed-point
+ * @param[out] exponent exponent
+ */
+void floatToFixedPointAndExponent(float input, int &fixedpoint, int &exponent);
```

Comment:
I think floatToFixedPointAndExponent is a clear name, but what do you think about using convertFloatToFixedPointAndExponent? I believe a name that includes the action of 'convert' is more intuitive for understanding the function's behavior.

```suggestion
void convertFloatToFixedPointAndExponent(float input, int &fixedpoint, int &exponent);
```